### PR TITLE
Fixes for observed installation issues.

### PR DESCRIPTION
**Pull request by davidjwbbc**
Closes 5G-MAG/rt-5gms-application-server#39.

This PR fixes an issue with the creation of the OpenAPI bindings during some distribution or installation processes.
This also updates the documentation for the project to include new files, renamed files and to clarify the user privileges needed to run this with its default configuration.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -5,19 +5,23 @@ In this directory you will find files to assist with development and testing of
...
+- README.md          - Project README file.
+- pyproject.toml     - The Python project description for building and installing the application.
+- build_scripts/     - Scripts used when building the python project.
   - backend.py - build backend wrapper to trigger extra build actions.
-  - generate_openapi - Will generate the OpenAPI python modules if not already present.
-- docs/          - Development documentation and examples.
+  - generate_5gms_as_openapi - Will generate the OpenAPI python modules if not already present.
+- docs/              - Development documentation and examples.
   - README.md - This document.
   - chc.json - An example ContentHostingConfiguration JSON file which provisions a pull-ingest AS for "Big Buck Bunny".
```
Comment (rjb1000) [inline docs/README.md:17]: Shouldn't `chc.json` be removed now that the example is available from rt-common-shared/5gms?

**Review by rjb1000 — APPROVED** [review]
Thanks, @davidjwbbc.

Looks good apart from the above comment about removing `chc.json` from `README.md`.

Let's see what @dsilhavy and @jordijoangimenez think.

File: docs/README.md (hunk 1 above)
Comment (jordijoangimenez) [reply docs/README.md:17]: There's a file inside docs/ which is not on the list? example-application-server.conf
the json has been moved to external/rt-commom-shared/..., isn't it?

Comment (jordijoangimenez) [reply docs/README.md:17]: These are the two things I've fixed directly:
https://github.com/5G-MAG/rt-5gms-application-server/issues/39#issuecomment-1283840836

**Comment by dsilhavy** [comment]
Changes looks good to me, one observation: I am still getting a ` No module named 'rt_5gms_as.openapi_5g'`. As pointed out in #39 Linux works for me though so it might also be a problem on my machine. We would need someone with a Mac device to try as well.

**Comment by davidjwbbc** [comment]
I wonder if there's a problem not being reported in the generator script. You could try running `build_scripts/generate_5gms_as_openapi` to see the error messages.

File: docs/README.md (hunk 1 above)
Comment (davidjwbbc) [reply docs/README.md:17]: Absolutely

**Comment by rjb1000** [comment]
Blocked awaiting approval from @dsilhavy and @jordijoangimenez.

Has @davidjwbbc addressed your feedback satisfactorily? If so, please hit the Approve button.

**Comment by jordijoangimenez** [comment]
I believe this looks good. Daniel, if you are happy please go ahead.

**Review by jordijoangimenez — APPROVED** [review]

**Review by dsilhavy — APPROVED** [review]

**Pull request merged**